roles/argocd/tasks/main.yaml

```diff
@@ -39,31 +38,41 @@
   when: argocd_wait_for_deployments | bool
 
 - name: Configure ArgoCD VirtualService
-  community.kubernetes.k8s:
+  kubernetes.core.k8s:
     state: present
     template: templates/argocd-vs.yaml.j2
   when: argocd_create_virtualservice | bool
 
 - name: Configure ArgoCD Project
-  community.kubernetes.k8s:
+  kubernetes.core.k8s:
     state: present
     template: templates/project.j2
 
 - name: Configure ArgoCD Applications
-  community.kubernetes.k8s:
+  kubernetes.core.k8s:
     state: present
     template: templates/application.j2
   loop: "{{ gitops_namespaces }}"
   loop_control:
     loop_var: loop_namespace
 
+- name: Configure ArgoCD git repository
+  kubernetes.core.k8s:
+    state: present
+    template: templates/argocd-repository-secret.yaml.j2
+
+- name: Update ArgoCD ConfigMap
+  kubernetes.core.k8s:
+    state: present
+    template: templates/argocd-configmap.yaml.j2
+
 - name: Configure Seldon Deploy git-creds secret
-  community.kubernetes.k8s:
+  kubernetes.core.k8s:
```

Comment:
Out of scope of this PR, but this is very Seldon Deploy specific? Thought that with ansible collections, we are aiming to keep these components as general-purpose as possible?

Reply:
Also, looking into this a bit deeper, I'm not sure how this secret is used (searching for `seldon_deploy_git_creds_secret_name` as defined in the defaults, I couldn't find any reference to it).

Reply:
It is used by Seldon Deploy. 
Happy to remove it from here once we add it to internal role.